megamek/src/megamek/client/generator/TeamLoadoutGenerator.java

```diff
@@ -652,15 +652,62 @@ public static ReconfigurationParameters generateParameters(
 
     // region Imperative mutators
     private static void setACImperatives(Entity e, MunitionTree mt, ReconfigurationParameters rp) {
+        setAC5Imperatives(e, mt, rp);
+        setAC10Imperatives(e, mt, rp);
         setAC20Imperatives(e, mt, rp);
     }
 
+    private static int getACWeaponCount(Entity e, String size) {
+        return (int) e.getWeaponList().stream()
+                .filter(w -> w.getName().toLowerCase().contains("ac") && w.getName().contains(size)).count();
+    }
+
+    private static int getACAmmoCount(Entity e, String size) {
+        return (int) e.getAmmo().stream()
+                .filter(w -> w.getName().toLowerCase().contains("ac") && w.getName().contains(size)).count();
+    }
+
+    // Set low-ammo-count AC5 carriers to use Caseless if ammo is <= 1/2 ton per tube
+    private static boolean setAC5Imperatives(Entity e, MunitionTree mt, ReconfigurationParameters rp) {
+        int ac5Count = getACWeaponCount(e, "5");
+
+        // TODO: remove this block when implementing new anti-ground Aero errata
+        // Ignore Aeros, which can't use most alt munitions, and those without AC5s.
+        if (e.isAero() || ac5Count == 0) {
+            return false;
+        }
+
+        // Always use Caseless if AC/5 ammo tons <= count of tubes
+        int ac5Ammo = getACAmmoCount(e, "5");
+        if (ac5Ammo <= ac5Count/2.0) {
+            mt.insertImperative(e.getFullChassis(), e.getModel(), "any", "AC/5", "Caseless");
+            return true;
+        }
+        return false;
+    }
+
+    // Set low-ammo-count AC10 carriers to use Caseless if ammo is <= 1 ton per tube
+    private static boolean setAC10Imperatives(Entity e, MunitionTree mt, ReconfigurationParameters rp) {
+        int ac10Count = getACWeaponCount(e, "10");
+
+        // TODO: remove this block when implementing new anti-ground Aero errata
```

Comment:
I know @Algebro7 is implementing some of the new Aero errata, but I don't know if they're doing the anti-ground errata. Tagging them here so that they're aware of this TODO (in case there is any cross-over).

Reply:
Side note: anyone working on this will also need to touch the other MM, MML, and MHQ ammo validation code - it's not a small lift.